src/app/PCM/templates/tpl-detail-view/tpl-property-detail-view/index.js

```diff
@@ -78,9 +83,23 @@ const TemplatePropertyDetailView = ({ mappedPcm }) => {
 
   //PRI_IMAGES is now used for the multi image display
   const imageCode = findCode('PRI_IMAGES') || findCode('PRI_IMAGE_URL')
+
   //Get suburb, state if both not null, otherwise just the one that isn't null
-  const location =
-    !!suburb && !!state ? `${suburb}, ${state}` : !!addressFull ? addressFull : suburb || state
+  const getLocation = () => {
+    let _suburb = suburb || addressSuburb
```

Comment:
What's the purpose of using _ infront of suburb. Are you trying to indicate that this is a instance variable?